openshift/android-sdk-persistent-template.json

```diff
@@ -25,7 +25,24 @@
         ],
         "resources": {
           "requests": {
-            "storage": "${VOLUME_CAPACITY}"
+            "storage": "${ANDROID_SDK_VOLUME_CAPACITY}"
+          }
+        }
+      }
+    },
+    {
+      "kind": "PersistentVolumeClaim",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${ANDROID_CONFIG_SERVICE_NAME}"
```

Comment:
the .android folder, which includes the default debug keystore file, temp build files, etc